app/services/sends_assignment_reminders.rb

```diff
@@ -1,11 +1,11 @@
 class SendsAssignmentReminders
-  def initialize
-    @week = Week.upcoming.first
-  end
-
   def call
-    Assignment.where(week: @week).find_each do |assignment|
-      UserMailer.assignment_reminder(assignment).deliver
+    Office.find_each do |office|
```

Comment:
Why do we need to iterate through these? Why can't we skip the office constraint and do it like before?

Reply:
Umm.... I think I need to do this because weeks are now weeks are created for one office now, so I can't just do it for one week anymore.

Reply:
Right!